app/services/PenaltiesFrontendService.scala

```diff
@@ -69,7 +69,7 @@ class PenaltiesFrontendService @Inject()() {
               )
               oldLPPDetails.copy(
                 metadata = LPPDetailsMetadata(
-                  mainTransaction = penaltyToCopy.get.lineItemDetails.get.head.mainTransaction,
```

Comment:
Can we scrap `combineLPPData` now?

Reply:
I think that goes a bit too far beyond the scope of the original ticket, I agree we can possibly scrap the `combineLPPData` function in some way but maybe that could be done when we handle manually removed LPPs which will require a rework of that logic